java/ql/automodel/publish.sh

```diff
@@ -1,17 +1,23 @@
 #!/bin/bash
 set -e
 
-# Add help message
+help="Usage: ./publish --[override-release] --[dry-run]
```

Comment:
```suggestion
help="Usage: ./publish [--override-release] [--dry-run]
```
I think this would be the standard syntax to denote optional arguments, right?